src/config.js

```diff
@@ -29,39 +29,31 @@ let cfgConfig = {};
 
 // Check if there is a user provided config file
 let cfgFile = path.resolve(nconf.get('configfile') || './.config.js');
-let cfgReason = '';
-const cfgFileExt = cfgFile.substr((cfgFile.lastIndexOf('.') + 1));
-if (cfgFile && fs.existsSync(cfgFile)) {
-  const cfgRead = fs.readFileSync(cfgFile, { encoding: 'utf-8' });
-  logger.info(`timings API config [${cfgFile}]:\n\n${cfgRead}\n`);
-  switch (cfgFileExt) {
-    case 'js':
-      cfgConfig = require(cfgFile);
-      break;
-    case 'json':
-      try {
+const cfgRead = fs.existsSync(cfgFile) ? fs.readFileSync(cfgFile, { encoding: 'utf-8' }) : '';
```

Comment:
yaml is a superset of json.  i also (without much context) suggest exiting if there's a problem loading the config file.  So, how about shortening to this:

```js
logger.info(`[timings API] - [CONFIG] using config [${configFile}]`);
const config = yaml.safeLoad(fs.readFileSync(configFile));
```